GameOfLife/Controller/GameManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace GameOfLife
+{
+    /// <summary>
+    /// Management of the game.
+    /// </summary>
+    public class GameManager
+    {
+        /// <summary>
+        /// Starts the game.
+        /// </summary>
+        public void Play()
+        {
+            UserInterface userInterface = new UserInterface();
+            GameView gameView = new GameView();
```

Comment:
use dependency injection for these `UserInterface userInterface = new UserInterface();
            GameView gameView = new GameView();`

Reply:
Changes have been made in Iteration 2 branch.